tests/publiccloud/migration.pm

```diff
@@ -66,11 +66,14 @@ sub run {
     # migration finished and instance rebooted
     record_info('INFO', 'Checking the migration succeed');
 
-    my $product_version = $instance->ssh_script_run(cmd => 'sudo cat /etc/os-release');
+    my $product_version = $instance->ssh_script_run(cmd => 'sudo cat /etc/os-release | grep ^VERSION_ID');
```

Comment:
this is just FYI message , why you cut it here ? we can keep full os-release content maybe it will be useful one day

Reply:
for me it makes sense to only product version in the variable.. So, many other details or there in os-release. (we can extract with appropriate variable later.)